src/visbook/visbook.interface.ts

```diff
@@ -0,0 +1,77 @@
+// TODO replace "unknown" for relevant properties
```

Comment:
There is a VS code extension which can help do this https://marketplace.visualstudio.com/items?itemName=MariusAlchimavicius.json-to-ts

Reply:
Wow, and here I was, typing everything by hand (pun intended...). Next time. ;)